docs/book/src/concepts.md

```diff
@@ -0,0 +1,109 @@
+kind: SecretProviderClass
+metadata:
+  name: my-provider
+spec:
+  provider: vault                             # accepted provider options: azure or vault or gcp
+  parameters:                                 # provider-specific parameters
+```
+
+> Refer to the provider docs for required provider specific parameters.
+
+Here is an example of a `SecretProviderClass` resource:
+
+```yaml
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: my-provider
+  namespace: default
+spec:
+  provider: azure
+  parameters:
+    usePodIdentity: "false"
+    useManagedIdentity: "false"
+    keyvaultName: "$KEYVAULT_NAME"
+    objects: |
+      array:
+        - |
+          objectName: $SECRET_NAME
+          objectType: secret
+          objectVersion: $SECRET_VERSION
+        - |
+          objectName: $KEY_NAME
+          objectType: key
+          objectVersion: $KEY_VERSION
+    tenantId: "$TENANT_ID"
+```
+
+Reference the `SecretProviderClass` in the pod volumes when using the CSI driver:
+
+```yaml
+volumes:
+  - name: secrets-store-inline
+    csi:
+      driver: secrets-store.csi.k8s.io
+      readOnly: true
+      volumeAttributes:
+        secretProviderClass: "my-provider"
+```
+
+> NOTE: The `SecretProviderClass` needs to be created in the same namespace as the pod.
+
+## SecretProviderClassPodStatus
+
+The `SecretProviderClassPodStatus` is a namespaced resource in Secrets Store CSI Driver that is created by the CSI driver to track the binding between a pod and `SecretProviderClass`. The `SecretProviderClassPodStatus` contains details about the current object versions that have been loaded in the pod mount.
+
+The `SecretProviderClassPodStatus` is created by the CSI driver in the same namespace as the pod and `SecretProviderClass` with the name `<pod name>-<namespace>-<secretproviderclass name>`.
+
+Here is an example of a `SecretProviderClassPodStatus` resource:
+
+```yaml
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClassPodStatus
+metadata:
+  creationTimestamp: "2021-01-21T19:20:11Z"
+  generation: 1
+  labels:
+    internal.secrets-store.csi.k8s.io/node-name: kind-control-plane
+    manager: secrets-store-csi
+    operation: Update
+    time: "2021-01-21T19:20:11Z"
+  name: nginx-secrets-store-inline-crd-dev-azure-spc
+  namespace: dev
+  ownerReferences:
+  - apiVersion: v1
+    kind: Pod
+    name: nginx-secrets-store-inline-crd
+    uid: 10f3e31c-d20b-4e46-921a-39e4cace6db2
+  resourceVersion: "1638459"
+  selfLink: /apis/secrets-store.csi.x-k8s.io/v1alpha1/namespaces/dev/secretproviderclasspodstatuses/nginx-secrets-store-inline-crd
+  uid: 1d078ad7-c363-4147-a7e1-234d4b9e0d53
+status:
+  mounted: true
+  objects:
+  - id: secret/secret1
+    version: c55925c29c6743dcb9bb4bf091be03b0
+  - id: secret/secret2
+    version: 7521273d0e6e427dbda34e033558027a
+  podName: nginx-secrets-store-inline-crd
+  secretProviderClassName: azure-linux
```

Comment:
```suggestion
  secretProviderClassName: azure-spc
```
> The `SecretProviderClassPodStatus` is created by the CSI driver in the same namespace as the pod and `SecretProviderClass` with the name `<pod name>-<namespace>-<secretproviderclass name>`.

secretProviderClassName here should match the last part of the SecretProviderClassPodStatus's name